packages/hash/frontend/src/components/BlockLoader/BlockLoader.tsx

```diff
@@ -131,7 +189,11 @@ export const BlockLoader: FunctionComponent<BlockLoaderProps> = ({
       accountId,
       entityId: entityId ?? "entityId-not-yet-set", // @todo ensure blocks always get sent an entityId
       entityTypeId,
-      properties: entityProperties,
+      properties: prepareHashEntityProperties(
```

Comment:
This'll do it for the root entity (well, a text entity one level inside the root entity), but not if there is rich text in anything at different levels of nesting – appreciate that we might not yet be able to support the hook service operating on text entities which aren't in a very specific place, but we could run the transformation to plain string on all entities by moving this conversion into `convertApiEntityToBpEntity`, no?